src/components/calcite-popover/calcite-popover.stories.ts

```diff
@@ -56,70 +56,35 @@ export default {
-        <calcite-popover
-          ${boolean("dismissible", false)}
-          ${boolean("disable-flip", false)}
-          ${boolean("disable-pointer", false)}
-          reference-element="reference-element"
-          placement="${select("placement", calcite_placements, "auto")}"
-          offset-distance="${number("offset-distance", 6)}"
-          offset-skidding="${number("offset-skidding", 0)}"
-          ${boolean("open", true)}
-          text-close="${text("text-close", "Close")}"
-        >
-          ${contentHTML}
-        </calcite-popover>
-      </div>
-    `;
-};
-
-export const RTL = (): string => {
-  return `
-      <div dir="rtl">
-        ${referenceElementHTML}
-        <calcite-popover
-          ${boolean("dismissible", false)}
-          ${boolean("disable-flip", false)}
-          ${boolean("disable-pointer", false)}
-          reference-element="reference-element"
-          placement="${select("placement", calcite_placements, "auto")}"
-          offset-distance="${number("offset-distance", 6)}"
-          offset-skidding="${number("offset-skidding", 0)}"
-          ${boolean("open", true)}
-          text-close="${text("text-close", "Close")}"
-        >
-          ${contentHTML}
-        </calcite-popover>
-      </div>
-    `;
-};
-
-export const DarkMode = (): string => {
-  return `
-      <div>
-        ${referenceElementHTML}
-        <calcite-popover
-        class="calcite-theme-dark"
-          ${boolean("dismissible", false)}
-          ${boolean("disable-flip", false)}
-          ${boolean("disable-pointer", false)}
-          reference-element="reference-element"
-          placement="${select("placement", calcite_placements, "auto")}"
-          offset-distance="${number("offset-distance", 6)}"
-          offset-skidding="${number("offset-skidding", 0)}"
-          ${boolean("open", true)}
-          text-close="${text("text-close", "Close")}"
-        >
-          ${contentHTML}
-        </calcite-popover>
-      </div>
-    `;
-};
-
-DarkMode.story = {
-  parameters: { themes: themesDarkDefault }
-};
+export const Simple = stepStory(
+  (): string => html`
+    <div style="width: 400px;">
+      ${referenceElementHTML}
+      <calcite-popover
+        ${boolean("dismissible", false)}
+        ${boolean("disable-flip", false)}
+        ${boolean("disable-pointer", false)}
+        reference-element="reference-element"
+        placement="${select("placement", calcite_placements, "auto")}"
+        offset-distance="${number("offset-distance", 6)}"
+        offset-skidding="${number("offset-skidding", 0)}"
+        ${boolean("open", false)}
+        text-close="${text("text-close", "Close")}"
+      >
+        ${contentHTML}
+      </calcite-popover>
+    </div>
+  `,
+  createSteps("calcite-popover")
+    .snapshot("Default")
+    .click("#reference-element")
+    .snapshot("Open")
+    .executeScript(setKnobScript({ story: "components-popover--simple", knob: "dismissible", value: "true" }))
```

Comment:
What's the story (badumtsss 🥁 ) for cases that need rich data and won't be able to set value via an attribute? Would we add a specific story for that use case that may or may not have steps associated with it?

Reply:
If the data needs to change, then a separate story is the way to go. We don't want screener to be handling changing of data/content.